### PR TITLE
docs: added aws via flightcontrol docs #1249

### PR DESCRIPTION
**Pull request by ModupeD**
### ❓ Type of change

- [ x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This change adds a new deployment provider to Nitro: deploying to AWS via Flightcontrol. Flightcontrol helps you deploy your Nitro projects (Both static & SSR) to your own AWS account with little to no configuration.

### 📝 Checklist

- [x ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by ModupeD** [comment]
@pi0 could you please review this

**Comment by danielroe** [comment]
@ModupeD Just checking - is there a difference between this PR and the one already open?

**Comment by ModupeD** [comment]
@danielroe the other one is outdated and cannot be edited, so my boss asked me to create a new one

**Comment by danielroe** [comment]
Perfect - got it 👌

File: src/presets/flightcontrol.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { defineNitroPreset } from "../preset";
+
+export const flightControl = defineNitroPreset({
```
[the remark this answers is not in the document]
Comment (ModupeD) [reply src/presets/flightcontrol.ts]: Thank you

**Review by danielroe — APPROVED** [review]

**Comment by flybayer** [comment]
hey @pi0, gentle nudge on this one :)

**Comment by pi0** [comment]
Hi, @ModupeD thanks for reworking this doc PR, and sorry for the delay in merging. To ensure the quality of officially listed deployment providers, we usually do a deployment with [nitro-deploys](https://github.com/unjs/nitro-deploys) playground (which still have to do)

Respecting the original author's request I've applied your changes via https://github.com/unjs/nitro/pull/970/commits/e7936d94d4a896572614c8940e948c2d41e0f352 and will be merged via #970 once deploy confirmed.

Feel free to make any other PRs to the main and improve docs and preset ❤️ 

Cheers.

**Pull request closed without merging**